addons/armaos/functions/fnc_hackingCommands_init.sqf

```diff
@@ -0,0 +1,32 @@
+/**
+ * Initilizes all hacking commands defined in CfgHackingCommands.
```

Comment:
Probably can be abstracted, to work for all command related modules

Reply:
Yes, at the moment it produces a lot of overhead. I need a distinguishable feature to reduce the amount of code. I will think about that.